app/pages/CheckYourAnswersPage.scala

```diff
@@ -44,8 +44,12 @@ case object CheckYourAnswersPage extends Page {
     }.getOrElse(routes.JourneyRecoveryController.onPageLoad())
 
   private def alreadySubscribedRoute(answers: UserAnswers): Call =
-    answers.get(BusinessTypePage).map {
-      case Individual | SoleTrader => routes.IndividualAlreadyRegisteredController.onPageLoad()
-      case _ => routes.BusinessAlreadyRegisteredController.onPageLoad()
-    }.getOrElse(routes.BusinessAlreadyRegisteredController.onPageLoad())
+    answers.subscriptionDetails match {
+      case Some(details) =>
+        details.businessType.map {
+          case Individual | SoleTrader => routes.IndividualAlreadyRegisteredController.onPageLoad()
+          case _ => routes.BusinessAlreadyRegisteredController.onPageLoad()
+        }.getOrElse(routes.IndividualAlreadyRegisteredController.onPageLoad())
```

Comment:
Why do we go to IndividualAlreadyRegisteredController when businessType is missing?

Reply:
Changed to error page as it makes more sense.

Reply:
Error page is also not enough, as discussed on the call.